router/middleware/tracing.go

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package middleware
```

Comment:
Can we get a corresponding test file for these functions?

Reply:
done, thanks